prebuild.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env sh
+
+# add source to path
+export PYTHONPATH="$(pwd)/src"
+
+# start inmanta app to generate parser/lexer in place
```

Comment:
The following files are generated: `parser.out`, `parsetab.py`. I added the first to the manifest file but left out the second. It should be included by default because it is a `*.py` file, right?

Reply:
I think so.